services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/PermissionUpdate.kt

```diff
@@ -11,3 +11,25 @@ data class PermissionDto(
     @field:Schema(description = "HankeKayttaja ID") val id: UUID,
     @field:Schema(description = "New access level in Hanke") val kayttooikeustaso: Kayttooikeustaso,
 )
+
+data class NewUserRequest(
+    @field:Schema(
+        description = "The first name of the user",
+    )
+    val etunimi: String,
+    @field:Schema(
+        description = "The last name of the user",
+    )
+    val sukunimi: String,
+    @field:Schema(
+        description = "The email address of the user. The invitation will be sent to this address.",
+    )
+    val sahkoposti: String,
+    @field:Schema(
+        description = "The phone number of the user.",
+    )
+    val puhelinnumero: String,
+) {
+    fun toDomain(): HankekayttajaInput =
```

Comment:
toDomain may be a bit misleading. HankeKayttaja seems to be the actual domain object. Would it make sense to remove HankekayttajaInput and use this class instead (they hold the same data) in hankekayttajaservice?

Reply:
`HankekayttajaInput` is a domain object that's used in many places and this is a DTO, so tightly coupling them might lead to problems later. I'll rename this to `toHankekayttajaInput` to be explicit.